quotes.json

```diff
@@ -407,4 +407,7 @@
 "quote" : "If you can dream it, you can achieve it.",
 "name" : "Zig Ziglar"
 }
-]
+{
+"quote" : "Anything is Possible",
+"name": "Kevin Garnett"
+},
```

Comment:
Thanks for the PR, John! Unfortunately, there's some syntax errors in this. The ending bracket for the array was deleted by accident, so that will need to be added back in at the end. Also, there needs to be a comma after the previous quote object.